### PR TITLE
Show file title in thumbnail popup

### PR DESCRIPTION
**Pull request by fieldOfView**
This PR changes the title of the popup to be the name of the UFP file. The popup is made a bit smaller, because the thumbnail files are always created at 300x300 pixels.

I also took the liberty to add a description and remove a todo here and there.

### CODE REVIEW

**Comment by jneilliii** [comment]
Thanks. I was planning on doing the filename thing eventually. You saved me some work. I'll review you're changes today.

File: octoprint_ultimakerformatpackage/static/js/UltimakerFormatPackage.js [hunk 1]
```diff
@@ -7,16 +7,18 @@
 $(function() {
 	function UltimakerformatpackageViewModel(parameters) {
 		var self = this;
-		self.current_file = ko.observable('/static/img/tentacle-20x20.png');
+		self.thumbnail_url = ko.observable('/static/img/tentacle-20x20.png');
+		self,thumbnail_title = ko.observable('');
```
Comment (jneilliii) [inline octoprint_ultimakerformatpackage/static/js/UltimakerFormatPackage.js]: Shouldn't this be `self.thumbnail_title`?

**Review by jneilliii — CHANGES_REQUESTED** [review]
Small typo @fieldOfView.

**Comment by fieldOfView** [comment]
Thanks for fixing my typo!

**Pull request merged**